Hazel/src/Platform/Windows/WindowsInput.cpp

```diff
@@ -8,17 +8,17 @@ namespace Hazel {
 
 	Input* Input::s_Instance = new WindowsInput();
 
-	bool WindowsInput::IsKeyPressedImpl(int keycode)
+	bool WindowsInput::IsKeyPressedImpl(KeyCode keycode)
 	{
 		auto window = static_cast<GLFWwindow*>(Application::Get().GetWindow().GetNativeWindow());
-		auto state = glfwGetKey(window, keycode);
+		auto state = glfwGetKey(window, static_cast<uint32_t>(keycode));
```

Comment:
You are converting enum to uint32 then there is implicit conversion from uint32 to int32

Reply:
You're right, i should've payed attention to that.